index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Alzubair Protfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/de9dd03ce2.js" crossorigin="anonymous"></script>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <div class="header">
+    <header class="container">
+      <h1 class="navbar">My Logo</h1>
+      <nav class="button"><i class="fa-solid fa-bars"></i></nav>
+    </header>
+  </div>
+
+  <div class="section-2">
+    <img src="Microverse-Student-Project-1/img.png" alt="background-img">
+    <div class="paragraph">
+      <p class="first-text">I’m Alzubair Glad to see you!</p>
+      <p class="second-text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <a class="connect-button" href="#">LET’S CONNECT</a>
+      <div class="social-buttons">
+        <ul>
+          <li><i class="emoji emoji1 fa-brands fa-twitter"></i></li>
+          <li><i class="emoji fa-brands fa-linkedin"></i></li>
+          <li><i class="emoji fa-solid fa-m"></i></li>
+          <li><i class="emoji fa-brands fa-github"></i></li>
+          <li><i class="emoji fa-solid fa-hand-peace"></i></li>
```

Comment:
- [ ] Good Job on implementing the change. Please the list of icons uses a `<ul>` tag to group them, and `<li>` per icon; each icon has a link `<a>` tag. you forgot the `<a>`, Kindly add it.